apps/launcher/graphicspage.cpp

```diff
@@ -44,7 +45,8 @@ Launcher::GraphicsPage::GraphicsPage(Files::ConfigurationManager &cfg, Settings:
     connect(fullScreenCheckBox, SIGNAL(stateChanged(int)), this, SLOT(slotFullScreenChanged(int)));
     connect(standardRadioButton, SIGNAL(toggled(bool)), this, SLOT(slotStandardToggled(bool)));
     connect(screenComboBox, SIGNAL(currentIndexChanged(int)), this, SLOT(screenChanged(int)));
-
+    connect(distantTerrainCheckBox, SIGNAL(toggled(bool)), this, SLOT(slotDistantTerrainChanged(bool)));
+    viewingDistanceDoubleSpinBox->setMaximum(std::numeric_limits<float>::max());
```

Comment:
I'm not really sure if this is a good way of setting a maximum to the viewing distance spin box.